memory-db/src/lib.rs

```diff
@@ -354,6 +354,9 @@ where
 	}
 }
 
+// not no_std implementation requires that hasmap core
+// got its implementation in parity-util-mem
```

Comment:
This comment is unclear to me, can you rephrase it? Is it "The `no_std` implementation requires that `Hashmap` has an impl of `MallocSizeOf` in `parity-util-mem`" maybe?